src/main/java/uk/gov/hmcts/darts/audio/service/impl/AudioTransformationServiceImpl.java

```diff
@@ -95,32 +79,6 @@ public List<MediaEntity> getMediaMetadata(Integer hearingId) {
             .toList();
     }
 
-    @Override
-    public Optional<UUID> getMediaLocation(MediaEntity media, Integer containerLocationId) {
```

Comment:
author note:
this method was used in the past by this class but now is no longer used (only by tests). So I've removed it